src/polyfem/utils/MatrixCache.hpp

```diff
@@ -84,14 +85,14 @@ namespace polyfem::utils
 	private:
 		size_t size_;
 		StiffnessMatrix tmp_, mat_;
-		std::vector<Eigen::Triplet<double>> entries_;
-		std::vector<std::vector<std::pair<int, size_t>>> mapping_;
+		std::vector<Eigen::Triplet<double>> entries_; ///< contains global matrix indices and corresponding value
+		std::vector<std::vector<std::pair<int, size_t>>> mapping_; ///< maps row indices to column index/local index pairs
 		std::vector<int> inner_index_, outer_index_;
-		std::vector<double> values_;
+		std::vector<double> values_; ///< maps local index to corresponding value
```

Comment:
no, this contains the values

Reply:
I see... I will remove the comment.

Reply:
Is there any extra reference/documentation for the MatrixCache class? Are the other comments in this class accurate? I was having trouble understanding the different mappings/caches and how they are connected to each other.